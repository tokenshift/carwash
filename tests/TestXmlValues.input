<password>secret password</password>
This is the secret password.
<password><![CDATA[pa[ss<>wo]rd]]></password>
Also a pa[ss<>w]rd].
<password>&lt;password&gt;</password>
The final "<password>".
