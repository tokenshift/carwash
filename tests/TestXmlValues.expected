<password>********</password>
This is the ********.
<password><![CDATA[********]]></password>
Also a ********.
****************</password>
The final "********".

